### DISCUSSION TITLE
Clips and snapshots path

### DISCUSSION THREAD

**Discussion by joelbibeau** [blakeblackshear/frigate#1059] [Ask A Question]
where to change the path for saving clips and snapshot to an smb
I want save to an file sharing

**Comment by blakeblackshear** [comment]
You can mount whatever path you want to the folder in the container as shown in the docs. Just modify your docker command or compose file as needed. Others have reported that HassOS doesn't support this, so it doesn't work if you are using the addon.

**Reply by tpbillund** [reply]
> You can mount whatever path you want to the folder in the container as shown in the docs. Just modify your docker command or compose file as needed. Others have reported that HassOS doesn't support this, so it doesn't work if you are using the addon.

Hi
Will not hijack this question, but my question is a bit in the same thread.

If you here a NAS and run two containers nodeRED and frigate.

I can currently get the file name over in NodeRED.

So my question is now, what is the local full path to one's FriGate media / clips folder so I can view the image via browser locally?

**Reply by blakeblackshear** [reply]
The full path is whatever you decide to mount from your host in the docker container.

**Reply by joelbibeau** [reply]
Ok, 
Am on VIRTUALbox for test before deploye on my rpi3b+.
I understand your answer but am 1/2 newbi with.
I try many thing to mount but always issue, dont work.
Sorry for my bad english.  Frigate is very hot
Thank
